apps/server/src/lib/contacts-cache.ts

```
@@ -0,0 +1,49 @@
+      last: now,
+    });
+  }
+
+  await KV.put(keyForUser(userId), JSON.stringify(Array.from(map.values())));
+}
+
+export async function suggestContacts(userId: string, query: string, limit = 10) {
+  const contacts = await getContacts(userId);
+  const lower = query.toLowerCase();
+  const filtered = contacts.filter(
+    (c) => c.email.toLowerCase().startsWith(lower) || (c.name && c.name.toLowerCase().startsWith(lower)),
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)